OWML.ModHelper.Logging/Output.cs

```diff
@@ -0,0 +1,44 @@
+﻿using OWML.Common;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace OWML.ModHelper.Logging
+{
+    public static class Output
```

Comment:
I think this is the first static class in OWML. I don't hate statics, I just don't like them. If you have a good reason for it, I'll allow it ;)

Reply:
Really just because I only use it for static things, never for instantiating. And I don't really see why we'd want to instantiate Output?

Reply:
Statics are OK if they don't have state. But this one does: OwmlOutput can be null or not depending on whether or not CreateOutput is called. It's mostly problematic for testing but generally considered bad practice.